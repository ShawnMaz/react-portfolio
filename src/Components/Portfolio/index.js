import {useState} from 'react';
import Project from '../Project';

function Portfolio(){
    const [projects] = useState([
        {
            name:'Giglr',
            image:'Giglr.png',
            deployedUrl:'https://giglr.herokuapp.com/',
            githubRepo:'https://github.com/KeeveRW11/giglr',
            techStack:'MySQL, Express, Node & Bootstrap'
        }, 
        {
            name:'Undercurrents',
            image:'Undercurrents.png',
            deployedUrl:'https://helpvisa.github.io/undercurrents/',
            githubRepo:'https://github.com/helpvisa/undercurrents',
            techStack:'JavaScript, jQuery & Bulma'
        }, 
        {
            name:'Weather Dashboard',
            image:'weather-dashboard.png', 
            deployedUrl:'https://shawnmaz.github.io/weather-forecast/',
            githubRepo:'https://github.com/ShawnMaz/weather-forecast',
            techStack:'HTML, Bootstrap, JavaScript'
        }
    ]);
    return(
        <section>
            <Project  projects={projects}/>
        </section>
    );
}

export default Portfolio;