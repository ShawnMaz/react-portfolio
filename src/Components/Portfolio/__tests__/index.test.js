import {render, cleanup} from '@testing-library/react'
import '@testing-library/jest-dom';
import Portfolio from '..';

afterEach(cleanup);

describe('Portfolio component', () => {
    it('renders', () => {
        render(<Portfolio />);
    });

    it('matches snapshot', () => {
        const {asFragment} = render(<Portfolio/>);
        expect(asFragment()).toMatchSnapshot();
    });
});