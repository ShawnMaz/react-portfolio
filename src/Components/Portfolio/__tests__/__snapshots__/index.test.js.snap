// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Portfolio component matches snapshot 1`] = `
<DocumentFragment>
  <section
    class="d-flex flex-column align-items-center mt-3"
  >
    <h2>
      Portfolio
    </h2>
    <section
      class="container"
    >
      <div
        class="row"
      >
        <div
          class="col-md-12 col-lg-5 p-0 my-3 mx-auto card"
        >
          <img
            alt="..."
            class="img-fluid border border-1 border-dark border-opacity-25 rounded-2 shadow-lg false"
            src="Giglr.png"
          />
        </div>
        <div
          class="col-md-12 col-lg-5 p-0 my-3 mx-auto card"
        >
          <img
            alt="..."
            class="img-fluid border border-1 border-dark border-opacity-25 rounded-2 shadow-lg false"
            src="Undercurrents.png"
          />
        </div>
        <div
          class="col-md-12 col-lg-5 p-0 my-3 mx-auto card"
        >
          <img
            alt="..."
            class="img-fluid border border-1 border-dark border-opacity-25 rounded-2 shadow-lg false"
            src="weather-dashboard.png"
          />
        </div>
        <div
          class="col-md-12 col-lg-5 p-0 my-3 mx-auto card"
        >
          <img
            alt="..."
            class="img-fluid border border-1 border-dark border-opacity-25 rounded-2 shadow-lg false"
            src="coding-quiz.png"
          />
        </div>
        <div
          class="col-md-12 col-lg-5 p-0 my-3 mx-auto card"
        >
          <img
            alt="..."
            class="img-fluid border border-1 border-dark border-opacity-25 rounded-2 shadow-lg false"
            src="social-network-api.png"
          />
        </div>
        <div
          class="col-md-12 col-lg-5 p-0 my-3 mx-auto card"
        >
          <img
            alt="..."
            class="img-fluid border border-1 border-dark border-opacity-25 rounded-2 shadow-lg false"
            src="ecommerce-api.png"
          />
        </div>
      </div>
    </section>
  </section>
</DocumentFragment>
`;
